README.adoc

```diff
@@ -1,502 +1,527 @@
 always appreciated and have a high likelihood of being merged quickly. Please send them!
 
-However, if you want or feel the need to change JJWT's functionality or core code, please do not issue a pull request 
-without [starting a new JJWT discussion](https://github.com/jwtk/jjwt/discussions) and discussing your desired 
-changes **first**, _before you start working on it_.
+However, if you want or feel the need to change JJWT's functionality or core code, please do not issue a pull request
+without https://github.com/jwtk/jjwt/discussions[starting a new JJWT discussion] and discussing your desired
+changes *first*, _before you start working on it_.
 
-It would be a shame to reject your earnest and genuinely-appreciated pull request if it might not align with the 
+It would be a shame to reject your earnest and genuinely-appreciated pull request if it might not align with the
 project's goals, design expectations or planned functionality.  We've sadly had to reject large PRs in the past because
-they were out of sync with project or design expectations - all because the PR author didn't first check in with 
+they were out of sync with project or design expectations - all because the PR author didn't first check in with
 the team first before working on a solution.
 
-So, please [create a new JJWT discussion](https://github.com/jwtk/jjwt/discussions) first to discuss, and then we 
+So, please https://github.com/jwtk/jjwt/discussions[create a new JJWT discussion] first to discuss, and then we
 can see easily convert the discussion to an issue and then see if (or how) a PR is warranted.  Thank you!
 
-<a name="contributing-help-wanted"></a>
-#### Help Wanted
++++<a name="contributing-help-wanted">++++++</a>+++
+
+==== Help Wanted
 
-If you would like to help, but don't know where to start, please visit the 
-[Help Wanted Issues](https://github.com/jwtk/jjwt/labels/help%20wanted) page and pick any of the 
+If you would like to help, but don't know where to start, please visit the
+https://github.com/jwtk/jjwt/labels/help%20wanted[Help Wanted Issues] page and pick any of the
 ones there, and we'll be happy to discuss and answer questions in the issue comments.
 
-If any of those don't appeal to you, no worries! Any help you would like to offer would be 
-appreciated based on the above caveats concerning [contributing pull requests](#contributing-pull-requests). Feel free
-to [discuss or ask questions first](https://github.com/jwtk/jjwt/discussions) if you're not sure. :)
+If any of those don't appeal to you, no worries! Any help you would like to offer would be
+appreciated based on the above caveats concerning <<contributing-pull-requests,contributing pull requests>>. Feel free
+to https://github.com/jwtk/jjwt/discussions[discuss or ask questions first] if you're not sure. :)
 
-<a name="overview"></a>
-## What is a JSON Web Token?
++++<a name="overview">++++++</a>+++
```

Comment:
Does this mean that the anchor is still valid/retained?  I'm not yet familiar with this type of adoc syntax.

Reply:
Yes, both the named and generated anchors should be the same